src/Satooshi/Bundle/CoverallsV1Bundle/Command/CoverallsV1JobsCommand.php

```diff
@@ -83,6 +83,13 @@ protected function configure()
             array()
         )
         ->addOption(
+            'json_path',
+            '-o',
+            InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
```

Comment:
Sorry for the late, I was a little bit busy these days.

Yes @keradus .. you are right. I noted a problem in the test I wrote. It would be better if I followed the TDD steps ..

I believe it is better to redo the changes to keep a better history, instead of fix it in the next commit. What do you think?